indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -20,6 +20,20 @@
 @import 'partials/boxes';
 @import 'partials/icons';
 
+%rating-bullet {
```

Comment:
Just an idea, since the file is growing quite a bit. Maybe we could move all this review-related rules to `abstracts/_abstract_reviews` to keep them better organized.